gammapy/scripts/image_pipe.py

```diff
@@ -175,13 +222,102 @@ def make_maps(self, radius, bkg_norm=True):
             Disk radius in pixels for the significance map.
         bkg_norm : bool
             If true, apply the scaling factor to the bkg map
+        spectral_index : float
+            Assumed power-law spectral index
+        exposure_weighted_spectrum : bool
+            True if you want that the total excess / exposure gives the integrated flux
         """
         self.make_total_counts()
         self.make_total_bkg(bkg_norm)
         self.make_significance(radius)
+        self.make_total_excess()
+        self.make_total_exposure(spectral_index, exposure_weighted_spectrum)
 
     def make_psf(self):
         log.info('Making PSF ...')
 
-    def make_exposure(self):
-        log.info('Making EXPOSURE ...')
+    def make_total_excess(self):
+        """Compute excess between counts and bkg map."""
+        self.maps["total_excess"] = SkyMap.empty_like(self.empty_image)
+        excess = self.maps["total_counts"].data - self.maps["total_bkg"].data
+        self.maps["total_excess"].data = excess
+
+    def make_exposure(self, obs_id, spectral_index=2.3, exposure_weighted_spectrum=False):
+        """Compute the exposure map for one observation.
+
+        Excess/exposure will give the differential flux at the energy Eref at the middle of the self.energy_band
+
+        If exposure_weighted_spectrum is true, it will give the integrated flux over the self.energy_band
+
+        Exposure is define as fallow:
+
+        .. math ::
+
+            EXPOSURE = \int A(E) \phi(E) * time_{obs} \, dE
+
+        assuming a power law for the flux :math:`\phi(E) = \phi_{Eref} \times \frac{E}{E_ref}^{spectral_index}`
```

Comment:
The problem with the docstring is that the backslash has both a special meaning for Sphinx and for LaTeX.
So at the moment Sphinx does something with the backslashes `\` in your formulae and then LaTeX sees an incorrect expression and errors out.

The solution is to use a "raw" string for the docstring, i.e. add an "r" character at the front of the docstring.
See [here](http://stackoverflow.com/questions/2081640/what-exactly-do-u-and-r-string-flags-do-in-python-and-what-are-raw-string-l)

To debug your LaTeX formulae, running Sphinx repeatedly is annoyingly slow.
For that I use http://www.chachatelier.fr/latexit/ and only once it's correct there, I copy & paste the formula over into the Python docstring.